frontend/src/views/welcome/Login.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <AppRow>
+    <AppCol class="col3">
+      <section id="menu">
+        <ul>
+          <li><a href="#">What is new MSP Direct?</a></li>
+          <li><router-link :to="{ name: 'CredentialsInfo' }">What are new log-in Credentials</router-link></li>
+          <li><a href="https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/system-access/msp-direct" target="_blank">Transactions in MSP Direct?</a></li>
+          <li><a href="https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/identity-and-authentication-services" target="_blank">Get BC Services Card or BCeID</a></li>
+        </ul>
+      </section>
+    </AppCol>
+    <AppCol>
+      <section id="loginButtons">
```

Comment:
This page feels a bit empty without a header or something. We can maybe bring this up with the business. 
Maybe something like MSP Direct Login

Reply:
Agreed. I'll check with Shabari.